scripting/get5.sp

```diff
@@ -107,6 +107,7 @@ char g_FormattedTeamNames[MATCHTEAM_COUNT][MAX_CVAR_LENGTH];
 char g_TeamFlags[MATCHTEAM_COUNT][MAX_CVAR_LENGTH];
 char g_TeamLogos[MATCHTEAM_COUNT][MAX_CVAR_LENGTH];
 char g_TeamMatchTexts[MATCHTEAM_COUNT][MAX_CVAR_LENGTH];
+char g_TeamCoaches[MATCHTEAM_COUNT][64];
```

Comment:
I believe it's possible for a team to have multiple coaches (i.e. coach + analyst) - I haven't verified that anytime lately though.

Reply:
Yeah I think this still may be the case, I'll fix this and update it to be an array in coming days!